### PR TITLE
Feature/284 refactor task unit

### LINKED ISSUE #284: Refactor (multiple choice) tasks
## Acceptance criteria (for teacher's view):

- [x] Buttons (Add, +, -) are rearranged
- [x] Localization is consistent (i.e. everything in English for now)
- [x] Template for task is added
- [x] Each answer is removable

## Acceptance criteria (for student's view):

- [x] Title and description are properly displayed

## Additional info:
Related issues: #162, #350

### PR DESCRIPTION
**Pull request by micpah**
## Description:
Implemented a few changes in task unit (see improvements below) as discussed during our last demo session.

Closes #284 

## Improvements
- Each task unit initially gets one task/question
- Each question initially gets two answers
- Each answer can be removed (not last one anymore)
- Deletion of task/question is revised

## Known Issues:
_NONE_

------
_Remember to prefix your PR-Title with `⚠ WIP: ` if you still work on it.
If you have reached a final state, remove the prefix._

### CODE REVIEW

File: app/webFrontend/src/app/unit/unit-form/task-unit-edit/task-unit-edit.component.scss [hunk 1]
```diff
@@ -22,20 +26,20 @@
...
-        //margin-left: 14px;
-        margin-right: 6px;
-        margin-top: 18px;
+        margin: 1rem;
       }
       .mat-form-field {
         flex-grow: 1;
       }
+      .remove {
+        min-width: min-content;
```
Comment (felixbrucker) [inline app/webFrontend/src/app/unit/unit-form/task-unit-edit/task-unit-edit.component.scss]: Has limited browser support, doesn't work in latest Firefox. Additionally it doesn't seem to affect appearance? (tested with Chrome)

**Comment by felixbrucker** [comment]
before:
image

after:
image

File: app/webFrontend/src/app/unit/unit-form/task-unit-edit/task-unit-edit.component.scss (hunk 1 above)
Comment (micpah) [reply app/webFrontend/src/app/unit/unit-form/task-unit-edit/task-unit-edit.component.scss]: It did affect appearance: padding was wider without it. Can also be set to e.g. 'unset' though.

**Comment by micpah** [comment]
Implemented requested change of @grosspersky

**Pull request merged**